src/services/TwoFactorService.ts

```diff
@@ -0,0 +1,116 @@
+import { authenticator } from 'otplib';
+import * as QRCode from 'qrcode';
+import { User } from '../entities/User';
+import { initORM } from '../db';
+
+class TwoFactorService {
+  private readonly appName = 'Ghost Drive';
+
+  /**
+   * Generate a new 2FA secret for a user
+   */
```

Comment:
create a function with 1 line of code then call this function only one time in all of this file wtf?